external-crates/move/crates/move-compiler/src/command_line/compiler.rs

```diff
@@ -317,17 +316,43 @@ impl Compiler {
             Some(p) => p,
             None => VfsPath::new(PhysicalFS::new("/")),
         };
+        let mut vfs_to_original_path = HashMap::new();
 
         let targets = targets
             .into_iter()
-            .map(|p| Ok(p.to_vfs_path(&vfs_root)?))
+            .map(|p| {
+                let original = p.path;
+                let vfs = p.to_vfs_path(&vfs_root)?;
+                vfs_to_original_path.insert(Symbol::from(vfs.path.as_str()), original);
+                Ok(vfs)
+            })
             .collect::<Result<Vec<_>, anyhow::Error>>()?;
 
         let mut deps = deps
             .into_iter()
-            .map(|p| Ok(p.to_vfs_path(&vfs_root)?))
+            .map(|p| {
+                let original = p.path;
+                let vfs = p.to_vfs_path(&vfs_root)?;
+                vfs_to_original_path.insert(Symbol::from(vfs.path.as_str()), original);
+                Ok(vfs)
+            })
             .collect::<Result<Vec<_>, anyhow::Error>>()?;
 
+        for path in targets.iter().chain(&deps) {
+            if !path.path.is_file().unwrap_or(false) {
+                debug_assert!(
```

Comment:
Do we need the assertion since we are bailing anyways?

Reply:
I want the assertion so we get a panic instead of an Err in debug builds